wimm.Secundatives.UnitTests/Extensions/Expect_Test.cs

```diff
@@ -82,6 +82,16 @@ public async Task ExpectTaskMessage_NoValue_ThrowsWithMessage()
             Assert.Equal(_message, ex.Message);
         }
 
+        [Fact]
+        public async Task ExpectTaskMessage_Value_ThrowsWithMessage()
```

Comment:
This was very disorienting... shouldn't it be `ExpectTaskMessage_Value_ReturnsValue` and 

```
Assert.Equal(10, await underTest.Expect(_message);
```

?

Reply:
Yes. Brain dead while finishing this up. good catch